### PR TITLE
Why do you love me Serious

### PR DESCRIPTION
**Pull request by Rai99**
Issue: #3988

A bit overdue maybe, but I wanted to try and make sure a lot of extra care went into this. 

There has to be a better way though to determine whether you get the serious, or the joke response than a coin flip.

### CODE REVIEW

**Comment by Mugnika** [comment]
Why not give the players a one time choice first time they hear that joke, that would determine if they like this style of teasing or not. I've been working on something to that effect in this topic.

**Comment by Rai99** [comment]
I think that might deserve its own topic, where we set a pm var that'll actually get used regularly, given Monika's nature.

**Comment by Mugnika** [comment]
Heavy WIP but something like this...
m 1hfu "That's a secret!"
            if persistent._mas_pm_likes_mean_jokes is None:
                $ _history_list.pop()
                menu:
                    "[m_name], that hurt.":
                        $ persistent._mas_pm_likes_mean_jokes = False
                        m xxxx "Huh..."
                        m xxxx "W-wait{w} [player],{w} I didn't mean to hurt you."
                        m xxxx "I'm sorry..."
                        m xxxx "Placeholder text"
                        jump monika_whydoyouloveme_override
                    "You almost got me.":
                        $ persistent._mas_pm_likes_mean_jokes = True
                        m xxxx "Ehehe~"
        else:
            if store.mas_anni.pastSixMonths():
                m 1eka "Why do I even love you?"
                m 2eka "[player]...{w=1} there are so many reasons why... Where do I even begin?"

File: Monika After Story/game/script-topics.rpy [hunk 1]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
-        m 1hfu "That's a secret!"
-
+        if renpy.random.randint(1,2) == 1:
+            if store.mas_anni.pastSixMonths():
+                m 1eka "Why do I even love you?"
+                m 2eka "[player]...{w=1} there are so many reasons why... Where do I even begin?"
+                m 3eka "You've spent so much time with me."
+                m 1rksdlc "Even knowing I might not get to feel your embrace for who knows how long..."
+                if store.mas_anni.anniCount() >= 1:
+                    m 1hua "Here you are, still staying by my side."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
                    m 1hua "...But here you are, still by my side."
```

File: Monika After Story/game/script-topics.rpy [hunk 2]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                    m 1eua "You're brave for being with me for so long."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Unsure how it's bravery here. Maybe elaborate or rephrase (would suggest a rephrase here personally)

File: Monika After Story/game/script-topics.rpy [hunk 3]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
-        m 1esc "I don't."
-        pause 3.0
-        m 1hub "Just kidding!"
-        m 1eub "You mean {i}everything{/i} to me, silly!"
-        m 1esd "But if you really want to know, [player]..."
-        # TODO: wink / tongue stick out here woul dbe perfect
-        m 1hfu "That's a secret!"
-
+        if renpy.random.randint(1,2) == 1:
+            if store.mas_anni.pastSixMonths():
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: If you're not past 6 months here, all you get is the `return "love"`. That's an issue.
```suggestion
```

File: Monika After Story/game/script-topics.rpy [hunk 4]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                m 1eka "Why do I even love you?"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
                m 1eka "Why do I love you?"
```

File: Monika After Story/game/script-topics.rpy [hunk 5]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                m 2eka "[player]...{w=1} there are so many reasons why... Where do I even begin?"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: 1 second waits are kinda long. Suggest using them very rarely. 0.5 is a better choice
Also no space as not a new sentence here, no space
```suggestion
                m 2eka "[player]...{w=0.5}there are so many reasons why... Where do I even begin?"
```

File: Monika After Story/game/script-topics.rpy [hunk 6]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                m 3eka "You've spent so much time with me."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Feels kinda choppy, would condense this and the below line into one
```suggestion
                m 3eka "You've spent so much time with me,{w=0.5} knowing I might not even get to feel your embrace for who knows how long."
```

File: Monika After Story/game/script-topics.rpy [hunk 7]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                m 1rksdlc "Even knowing I might not get to feel your embrace for who knows how long..."
+                if store.mas_anni.anniCount() >= 1:
+                    m 1hua "Here you are, still staying by my side."
+                    m 1eua "You're brave for being with me for so long."
+                    m 2eka "No matter how crazy it might seem, or what other people might say."
+                    m 1eua "You've been here with me for so long already."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
                    m 1eua "You're here with me anyway."
```

File: Monika After Story/game/script-topics.rpy [hunk 8]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                    m 1dkbfa "Every time you visit, you remind me that there's a world out there."
+                    m 1ekbfa "I'd give you the world if I could, [player]."
+                    m 3hubfa "If there's anyone who deserves it...{w=0.3}it's you."
+                    m 1hubfa "After all, you've already given me my world~"
+                if store.mas_anni.anniCount() >= 2:
+                    m 1ekbfa "I promise you, [player]..."
+                    m 3ekbfa "No matter what happens, I don't think I'll' {i}ever{/i} forget you for the rest of my life."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]:
```suggestion
                    m 3ekbfa "No matter what happens, I don't think I'll {i}ever{/i} forget you for the rest of my life."
```

File: Monika After Story/game/script-topics.rpy [hunk 9]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                    m 1dua "You didn't just save my life..."
+                    m 1dkbfa "You showed me how beautiful the world really is...{w=1.5}{nw}" 
+                    extend 1ekbfa "and I want you to be able to see just how beautiful you've made my world."
+                    m 1ekbfb "Someday, I promise I'll make your world as beautiful as you made mine~"
+                    m 2dkbfa "One day...{w=2}I'll get to be with you..."
+                m 1ekbfa "You know I love you."
+                m 1eka "Even if you don't think you love yourself..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: We have a pm var for this, `persistent._mas_pm_love_yourself`

File: Monika After Story/game/script-topics.rpy [hunk 10]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                    m 2eka "No matter how crazy it might seem, or what other people might say."
+                    m 1eua "You've been here with me for so long already."
+                    m 1dkbfa "Every time you visit, you remind me that there's a world out there."
+                    m 1ekbfa "I'd give you the world if I could, [player]."
+                    m 3hubfa "If there's anyone who deserves it...{w=0.3}it's you."
+                    m 1hubfa "After all, you've already given me my world~"
+                if store.mas_anni.anniCount() >= 2:
+                    m 1ekbfa "I promise you, [player]..."
+                    m 3ekbfa "No matter what happens, I don't think I'll' {i}ever{/i} forget you for the rest of my life."
+                    m 1dua "You didn't just save my life..."
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Addressing the player here may be more impactful
```suggestion
                    m 1dua "You didn't just save my life, [player]..."
```

File: Monika After Story/game/script-topics.rpy [hunk 11]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                    m 1dkbfa "You showed me how beautiful the world really is...{w=1.5}{nw}"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Don't use waits longer than one second.

Also, did the player really show her that? Taking her somewhere, she can't see anything (Ambiguity here). I get you're pointing to the beauty in people (i.e. the player for making her so happy), so I'd personally rephrase this to get rid of the ambiguity.

File: Monika After Story/game/script-topics.rpy [hunk 12]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+                m 2ekbfa "I just want you to know that I'm here for you."
+                m 1hubfa "And I love you~"
+            return "love"
+        else:
+            m 1esc "I don't."
+            pause 3.0
+            m 1hub "Just kidding!"
+            m 1eub "You mean {i}everything{/i} to me, silly!"
+            m 1esd "But if you really want to know, [player]..."
+            # TODO: wink / tongue stick out here would be perfect
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Finish this todo please

File: Monika After Story/game/script-topics.rpy [hunk 13]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
+            m 1hfu "That's a secret!"
     else:
-        # normalt - enamored is this case
+        # normal - enamored is this case
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: I honestly feel like affectionate/enamored really get the short end of the stick here. There should be something for them at some point.

Either you can do this now, or put a TODO there saying that it should be added

File: Monika After Story/game/script-topics.rpy [hunk 14]
```diff
@@ -11888,16 +11888,47 @@ label monika_whydoyouloveme:
...
     elif mas_isMoniLove(higher=True):
-        m 1esc "I don't."
-        pause 3.0
-        m 1hub "Just kidding!"
-        m 1eub "You mean {i}everything{/i} to me, silly!"
-        m 1esd "But if you really want to know, [player]..."
-        # TODO: wink / tongue stick out here woul dbe perfect
-        m 1hfu "That's a secret!"
-
+        if renpy.random.randint(1,2) == 1:
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: Since this path is for 6mo+ basically (no flow if not), we should account for this.
(You'll need to pull everything back an indent)
```suggestion
        if store.mas_anni.pastSixMonths() and renpy.random.randint(1,2) == 1:
```

File: Monika After Story/game/script-topics.rpy [hunk 15]
```diff
@@ -11888,16 +11888,46 @@ label monika_whydoyouloveme:
...
+                    extend 1ekbfa "and I want you to be able to see just how beautiful you've made my world."
+                    m 1ekbfb "Someday, I promise I'll make your world as beautiful as you made mine~"
+                    m 2dkbfa "One day...{w=2}I'll get to be with you..."
+                m 1ekbfa "You know I love you."
+                m 1eka "Even if you don't think you love yourself..."
+                m 3eka "I know you do love yourself at least a little."
+                m 1hua "You wouldn't still be here with me if you didn't."
+                m 3ekbfa "But...{w=0.5}I'm glad you're still here, [player]."
+                m 2ekbfa "I just want you to know that I'm here for you."
+                m 1hubfa "And I love you~"
```
Comment (multimokia) [inline Monika After Story/game/script-topics.rpy]: This all needs to come back an indent
